### ISSUE TITLE
Docs: Formatting issue with long strings

### ISSUE THREAD

**Issue by noahmulfinger** [Esri/arcgis-rest-js#96]
The property table on the doc pages expands too far when there's a long string without a word break (see the sampleserver url below):
![screen shot 2017-12-28 at 11 06 09 am](https://user-images.githubusercontent.com/6539297/34420579-40f2ec3c-ebbf-11e7-9823-add6667d4ebb.png)

**Comment by patrickarlt** [comment]
Here is some sample css from the developers site that probably fixes this. Just swap `.usertext` for `code` and make this apply to all tables.

```css
/** only apply to tables inside <main> for sanity **/
main table {
  table-layout: fixed; /** force table to evenly distribute column widths or by <colgroup> widths **/
}

main table colgroup:first-child,
main table tr td:first-child, 
main table tr th:first-child  {
  width: 33% !important; /** force the first column to 33% width **/
}

/** only apply to usertext inside tables inside <main> for sanity **/
main table .usertext {
  overflow: auto; /** show a scrollbar automatically**/
  display: inline-block; /**  display span as a block **/
  white-space: nowrap; /** do not wrap text at all **/
  box-sizing: border-box;
  max-width: 97%; /** fallback width for non -calc browser **/
  max-width: calc(100% - .25rem);  /** make width slightly smaller then column **/
  box-shadow: 1px 0 0px #efefef, 0px 1px 0px #efefef, -1px 0 0px #efefef, 0px -1px 0px #efefef; /** Fake a border around the scrollbar **/
  border-radius: 0;  /** remove border radius to clear scrollbar **/
  vertical-align: -10px;
  border: none;
}

/** only apply to cells inside tables inside <main> for sanity **/
main table td {
  vertical-align: top; /** align text to top of table cells rather then middle**/
}

/** only apply to cells inside tables inside <main> for sanity **/
main table .stentry .usertext {
  width: 100%; /** user text can be 100% in simple tables **/
}
```

**Comment by patrickarlt** [comment]
Calcite Web issue https://github.com/Esri/calcite-web/issues/888.

**Comment by jgravois** [comment]
the problem isn't isolated to `<code>` blocks. proposed fix (from a certified css hack) in #163

**Comment by jgravois** [comment]
resolved in https://github.com/Esri/arcgis-rest-js/pull/531
